crates/node/src/network_server/metrics.rs

```diff
@@ -188,28 +188,35 @@ pub async fn render_metrics(State(state): State<NodeCtrlHandlerState>) -> String
     let manager = RocksDbManager::get();
     let all_dbs = manager.get_all_dbs();
 
+    let mut global_labels: Vec<String> = vec![];
+    for (k, v) in state.prometheus_handle.global_labels() {
+        global_labels.push(format!("{}=\"{}\"", k, formatting::sanitize_label_value(v)));
+    }
```

Comment:
Instead of recreating these labels on every `render_metrics` call, can we calculate them once and then reuse? The `prometheus_handle` could host it, for example.